packages/snap-store-mobx/src/Search/Stores/SearchFacetStore.ts

```diff
@@ -100,7 +100,8 @@ export class Facet {
 		services: StoreServices,
 		storage: StorageStore,
 		facet: SearchResponseModelFacetValue | SearchResponseModelFacetRange,
-		facetMeta: MetaResponseModelFacet
+		facetMeta: MetaResponseModelFacet,
+		config: SearchStoreConfig | AutocompleteStoreConfig
```

Comment:
I think it makes more sense to pass FacetStoreConfig instead of the full store config here. It is already defined on line 49 above,  but would require some refactoring to get the non facet-specific configuration and default values set.